torch/lib/TH/THGenerateComplexTypes.h

```diff
@@ -188,8 +187,8 @@ typedef double _Complex zx;
 
 // #pragma message "NOW DOING COMPLEX double"
 
-#define real zx
-#define accreal zx
```

Comment:
Sure, I'm reading your cuda part. I guess I can join you in a few days. And I'll probably touch THNN first. I will probably need a complex network recently XD...